dll/win32/kernel32/wine/resource.c

```diff
@@ -51,7 +51,9 @@ static __inline BOOL set_ntstatus( NTSTATUS status )
 DEBUG_CHANNEL(resource);
 
 /* we don't want to include winuser.h just for this */
+#ifndef IS_INTRESOURCE
 #define IS_INTRESOURCE(x)   (((ULONG_PTR)(x) >> 16) == 0)
+#endif 
```

Comment:
For the final version of this PR I would suggest that these respective build fixes get moved to their corresponding original commits where they would make sense.